docs/application/dotnet/guides/machine-learning/pipeline.md

```diff
@@ -174,6 +174,127 @@ If you need to stop the data stream or select the data flow with multiple stream
     pipe.Start();
     ```
 
+## Custom Filter
+
+For your convenience, NNStreamer provides an interface for processing the tensor data with the `custom-easy` framework. After registering the user-defined callback method with the input and the output tensor information, NNStreamer can manipulate tensor data in the pipeline without an independent shared object. Since the callback method works as **filter** in the pipeline, it is named as `Custom Filter`.
+
+Note that the Custom Filter on the dotnet layer shows relatively lower performance than those of the native layer because of marshaling and unmarshalling between the dotnet and native layer. If your application is mission-critical, then use the native Custom Filter.
```

Comment:
Previous line Custom Filter is highlighted in code tag, should that be followed throughout? Or was it to introduce the term>

Reply:
The previous 'Custom Filter' is for the introduction term and it is just highlighted.

Reply:
@mijin85cho, looking for your opinion here.

Reply:
As @again4you replied, previous was to highlight for the introduction. I don't think we need to emphasize every supported function name.